### PR TITLE
Capstone Exercise - AfriCities Summit 2022

### PR DESCRIPTION
**Pull request by WalterOkumu**
- Set up linters
- Replicated design from guidelines
- Customized page as per AfriCities Summit site
- Added images from past events

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,7 +82,7 @@ <h5>&quot;Karibu Kisumo!!!&quot;</h5>
         <h1>The 9th AfriCities Summit 2022</h1>
```
Comment (VICTOR-CHUKWU) [inline index.html:82]: - [ ] as your are allowed to have a different content from the given design, the project require that you stick to color and font size, Kindly fix the color and font size of you `h1` tag.
  your design :point_down: 
Screenshot from 2022-08-02 22-37-43
project design :point_down: 
Screenshot from 2022-08-02 22-42-10

File: styles/sass/main.scss [hunk 2]
```diff
@@ -544,592 +571,599 @@ body {
...
+                width: 50px;
+                height: 50px;
+              }
+            }
 
-      .program-container {
-        display: flex;
-        padding: 30px;
+            .title {
+              width: 35%;
```
Comment (VICTOR-CHUKWU) [inline styles/sass/main.scss]: - [ ] Kindly align your card title by making the width bigger than 35%, it currently looks not too good
Screenshot from 2022-08-02 22-44-21

File: about.html [hunk 3]
```diff
@@ -59,7 +59,7 @@
 
       <div class="desktop-navbar">
```
Comment (VICTOR-CHUKWU) [inline about.html:60]: - [ ] Your about page destop view should have a dark background footer, kindly fix this
  
Screenshot from 2022-08-02 22-51-15

File: README.md [hunk 4]
```diff
@@ -1,2 +1,33 @@
...
+# Microverse Capstone Project - 1 - AfriCities Summit
+
+ A website showcasing the Africities Summit in Kisumu County, Kenya
+
+## Built With
+
+- Major languages - HTML & CSS
+- Preprocessor - SASS
+
+## Live Demo (if available)
```
Comment (VICTOR-CHUKWU) [inline README.md:12]: - [ ] Images in your live page is broken, kindly fix this
Screenshot from 2022-08-02 22-55-19

File: README.md [hunk 5]
```diff
@@ -1,2 +1,33 @@
...
+👤 **Walter Okumu**
+
+- GitHub: [@WalterOkumu](https://github.com/WalterOkumu)
+- Twitter: [@OkumuOriaro](https://twitter.com/OkumuOriaro)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (VICTOR-CHUKWU) [inline README.md:27]: - [ ] Your  README should contain an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin] the author of the original design.

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @WalterOkumu , 
## STATUS: CHANGES REQUIRED :recycle: 
Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mikethreels — APPROVED** [review]
Hi Okumu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**